components/scripts/lib/libs.sh

```diff
@@ -1,37 +1,40 @@
 #!/usr/bin/env bash
 
 # shellcheck source=lib/color.sh
-source "${LIB_DIR}/color.sh" || { echo "Couldn't find '${LIB_DIR}/color.sh'"; exit 1; }
+source "${LIB_DIR}/color.sh" || { echo "Couldn't find '${LIB_DIR}/color.sh'"; exit 2; }
```

Comment:
Originally, I was thinking we can't use `die` here because it hasn't been loaded yet. And we can't load `exit-code.sh` first because it uses constants from `color.sh`.

Although, now that I've thought about it, I know how to work around this. `exit-code.sh` can load `color.sh` directly. It also depends on a function from `info.sh` (`print_ln`), but it could also load that directly. I'll give that a try tomorrow (and update this file to use `die`).